monday/query_joins.py

```diff
@@ -356,6 +357,22 @@ def get_columns_by_board_query(board_ids):
         }''' % board_ids
 
 
+def duplicate_board_query(board_id: int, duplicate_type: DuplicateTypes):
```

Comment:
can you also add a hint (`str`) for the return type?

Reply:
Oh, yes. Sure! It makes sense